### PR TITLE
Catch OSError when getting file mtime & ctime

### PR DESCRIPTION
**Pull request by takluyver**
Closes gh-2757

### CODE REVIEW

File: notebook/services/contents/filemanager.py [hunk 1]
```diff
@@ -235,15 +235,18 @@ def _base_model(self, path):
         info = os.lstat(os_path)
         try:
             last_modified = tz.utcfromtimestamp(info.st_mtime)
-        except ValueError:
+        except (ValueError, OSError):
             # Files can rarely have an invalid timestamp
             # https://github.com/jupyter/notebook/issues/2539
+            # https://github.com/jupyter/notebook/issues/2757
             # Use the Unix epoch as a fallback so we don't crash.
+            self.log.warn('Invalid mtime %s for %s', info.st_mtime, os_path)
```
Comment (minrk) [inline notebook/services/contents/filemanager.py]: log.warning to avoid deprecation warnings (and below)

Comment (takluyver) [inline notebook/services/contents/filemanager.py]: Done. I always get that one wrong.

**Comment by takluyver** [comment]
@gnestor I've optimistically marked this for 5.1, but feel free to bump it to 5.2 if you'd prefer.

**Comment by gnestor** [comment]
I think this should be included in 5.1. Thanks @takluyver!

**Pull request merged**